pkg/server/http/ipfs.go

```diff
@@ -0,0 +1,184 @@
+package httpserver
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"os"
+	"path/filepath"
+	"strings"
+	"time"
+
+	lassie "github.com/filecoin-project/lassie/pkg/lassie"
+	"github.com/ipfs/go-cid"
+)
+
+func ipfsHandler(res http.ResponseWriter, req *http.Request) {
+	logger := NewRequestLogger(req.Method, req.URL.Path)
```

Comment:
why do we want a separate, custom logger here?

Reply:
It ended up not being an actual logger, but it prints terse server requests and responses to stdout. I've taken the stance that  printing is for command feedback, while logging is for application level information.